nucypher/characters/chaotic.py

```diff
@@ -132,15 +133,16 @@ class Felix(Character, NucypherTokenActor):
     TEMPLATE_NAME = 'felix.html'
 
     # Intervals
-    DISTRIBUTION_INTERVAL = 60*60   # seconds (60*60=1Hr)
-    DISBURSEMENT_INTERVAL = 24      # (24) hours
+    DISTRIBUTION_INTERVAL = 60  # seconds (60*60=1Hr)
+    DISBURSEMENT_INTERVAL = 24 * 365  # only distribute tokens to the same address once each year.
     STAGING_DELAY = 10              # seconds
 
     # Disbursement
     BATCH_SIZE = 10                 # transactions
-    MULTIPLIER = 0.95               # 5% reduction of previous stake is 0.95, for example
+    MULTIPLIER = Decimal('0.9')     # 10% reduction of previous stake is 0.9, for example
+                                    # this will result in 90 days of distribution
     MINIMUM_DISBURSEMENT = 1e18     # NuNits
-    ETHER_AIRDROP_AMOUNT = int(2e18)  # Wei
+    ETHER_AIRDROP_AMOUNT = 1e17     # Wei
```

Comment:
Just for other readers - this is `0.1 ETH`

Reply:
oh I guess I should put the int() back...